README.md

```diff
@@ -7,20 +7,20 @@ _Modern Firefox Telemetry for mobile platforms_
 
 ---
 
-**Note: This new approach for Glean is currently in development and not yet ready for use.
-The working and supported library is [Glean in android-components][glean-ac].**
+**Note: This new approach for the Glean SDK is currently in development and not yet ready for use.
+The working and supported library is the [Glean SDK in android-components][glean-ac].**
 
 ---
 
 ## Documentation
 
 All documentation is available online:
 
-## [The Glean Book][book]
+## [The Glean SDK Book][book]
```

Comment:
Can you please change this for all the other documentation files as well? For example, `contributing.md` still refers to `Glean` instead of `Glean SDK`.

Reply:
Actually, we can take that to a follow-up bug.